package.json

```diff
@@ -140,6 +140,13 @@
       "hidden": true
     },
     {
+      "name": "metadata.endpoint",
+      "title": "Local Metadata Endpoint",
+      "type": "string",
+      "value": "https://page-metadata-service.stage.mozaws.net/v1/metadata",
```

Comment:
Post merge comment, but do we now need to change this to the fresh, new prod stack URL?